src/layers/viv-viewer-layer/viv-viewer-layer.js

```diff
@@ -65,6 +73,39 @@ export class MicroscopyViewerLayer extends CompositeLayer {
     }
   }
 
+  _overrideChannelProps() {
```

Comment:
Not sure this Python convention carries over to ES?

Reply:
I have seen it in a fair number of repos - I am not sure this is convention in ES but I like it.